mathics/builtin/list/rearrange.py

```diff
@@ -398,7 +396,6 @@ def eval(self, expr, n, evaluation: Evaluation):
 
 class _SetOperation(Builtin):
     messages = {
-        "normal": "Non-atomic expression expected at position `1` in `2`.",
```

Comment:
Notice that in the duplication we have variation in the message: Non-atomic versus Nonatomic